WcaOnRails/app/mailers/avatars_mailer.rb

```diff
@@ -12,4 +12,17 @@ def notify_user_of_avatar_rejection(user, reason)
       subject: "Your avatar has been rejected",
     )
   end
+
+  def notify_user_of_avatar_removal(remover_user, user, reason)
+    @remover_user = remover_user
+    @user = user
+    @reason = reason
+
+    mail(
+      from: Team.wrt.email,
+      to: user.email,
+      reply_to: 'results@worldcubeassociation.org',
+      subject: "Your avatar has been removed by #{@remover_user.name}",
```

Comment:
Need some clarity:
Who is this email going to? WRT and The user who removed the avatar? Or wrt and the user whose avatar was removed?

I think wrt would prefer it just be internal (the person who removed it and wrt), in which case “your” in the subject should probably be changed to the wca id of the user  whose avatar was removed.

Reply:
Please remove remover_user.name from the subject and also do not include this information anywhere else!

WCA teams act as an entity, exposing actions of individual team members  to public entities should never happen if not strictly necessary.

Reply:
hi @Jambrose777 @SAuroux 

The email is going to the user whose avatar was removed and yes it includes the name of the user who removed the avatar.

If you check the issue description - https://github.com/thewca/worldcubeassociation.org/issues/7300#issue-1410442641, it clearly states `When an avatar is removed, an email notification to the user with who removed the avatar will be really helpful`.

Is this not the required behavior? I am confused now🤔.

Reply:
> Is this not the required behavior? I am confused now🤔.

Sorry about the confusion, I see how the description was misleading. But what I stated about individual team members definitely needs to be taken into account. Ideally, the notification should distinguish the following two cases:

Case 1: removal by a WRT member
- subject includes "by WRT"
- do **not** include 'results@worldcubeassociation.org' in CC
Case 2: removal by someone else 
- subject includes "by #{@remover_user.name}"
- include 'results@worldcubeassociation.org' in CC

Could you change the implementation accordingly? Thanks in advance!

Reply:
@SAuroux sure. Thanks for the clarification. I will add the changes👍

Reply:
Hi, @SAuroux @Jambrose777  I have pushed the changes. Can you please check?